src/QuantumSdk/Sdk/Sdk.props

```diff
@@ -23,43 +23,42 @@
-    <PackageReference Condition="'$(ResolvedQuantumIntrinsics)' == 'Type3'" Include="Microsoft.Quantum.Type3.Core" Version="0.18.2107151063-beta" IsTargetPackage='true' />
-    -->  
+    <PackageReference Condition="$(IncludeQSharpCorePackages)" Include="Microsoft.Quantum.QSharp.Core" Version="0.20.2110171573" />
+    <PackageReference Condition="$(IncludeQSharpCorePackages)" Include="Microsoft.Quantum.Standard" Version="0.20.2110171573" />
+    <!-- Provider packages included for specific execution targets. -->
+    <PackageReference Condition="$(EnableQirSubmission) And $(IncludeProviderPackages)" Include="Microsoft.Quantum.Providers.Core" Version="0.20.2110171573" />
+    <PackageReference Condition="'$(ResolvedProcessorArchitecture)' == 'HoneywellProcessor' And $(IncludeProviderPackages)" Include="Microsoft.Quantum.Providers.Honeywell" Version="0.20.2110171573" />
+    <PackageReference Condition="'$(ResolvedProcessorArchitecture)' == 'IonQProcessor' And $(IncludeProviderPackages)" Include="Microsoft.Quantum.Providers.IonQ" Version="0.20.2110171573" />
+    <PackageReference Condition="'$(ResolvedProcessorArchitecture)' == 'QCIProcessor' And $(IncludeProviderPackages)" Include="Microsoft.Quantum.Providers.QCI" Version="0.20.2110171573" />
+    <!-- Target packages included for specific execution targets. -->
+    <PackageReference Condition="'$(ResolvedQuantumIntrinsics)' == 'Type1' And $(ResolvedProcessorArchitecture.Contains('QirProcessor'))" Include="Microsoft.Quantum.Type1.Core" Version="0.20.2110171573" IsTargetPackage='true' />
+    <PackageReference Condition="'$(ResolvedQuantumIntrinsics)' == 'Type2' And $(ResolvedProcessorArchitecture.Contains('QirProcessor'))" Include="Microsoft.Quantum.Type2.Core" Version="0.20.2110171573" IsTargetPackage='true' />
+    <PackageReference Condition="'$(ResolvedQuantumIntrinsics)' == 'Type3' And $(ResolvedProcessorArchitecture.Contains('QirProcessor'))" Include="Microsoft.Quantum.Type3.Core" Version="0.20.2110171573" IsTargetPackage='true' />
     </ItemGroup>
 
   <!-- Include the qir generation and docs generation compiler extensions if needed. -->
   <ItemGroup>
     <PackageReference Condition="$(QSharpDocsGeneration)"
-                      Include="Microsoft.Quantum.DocumentationGenerator" Version="0.18.2107151063-beta"
+                      Include="Microsoft.Quantum.DocumentationGenerator" Version="0.20.2110171573"
                       IsImplicitlyDefined="true" 
                       IsQscReference="true" ExecutionTarget="Any" Priority="-1" PrivateAssets="All" />
-    <PackageReference Condition="$(QirGeneration) Or '$(ResolvedProcessorArchitecture)' == 'MicrosoftSimulator'"
-                      Include="Microsoft.Quantum.QirGeneration" Version="0.18.2107151063-beta"
+    <PackageReference Condition="$(QirGeneration) Or $(EnableQirSubmission)"
```

Comment:
It seems like this condition is failing during builds because when this is evaluated, `$(QirGeneration)` is still a string (`default`).

Reply:
I think you are right, the value for `QirGeneration` in src/QuantumSdk/DefaultItems/DefaultItems.props.v.template needs to be changed from `default` to `false`.

Reply:
The QirGeneration property is set to either true or false in the DefaultItems.targets file. The issue turned out to be that the .targets file wasn't value due to using Exclude without an Include.